pkg/minishift/cache/oc_caching.go

```diff
@@ -50,7 +49,12 @@ func (oc *Oc) GetCacheFilepath() string {
 }
 
 func (oc *Oc) isCached() bool {
-	if _, err := os.Stat(filepath.Join(oc.GetCacheFilepath(), OC_BINARY_NAME)); os.IsNotExist(err) {
+	binaryName := OC_BINARY_NAME
+	if runtime.GOOS == "windows" {
```

Comment:
Why do you do the check here? Why not set the _OC_BINARY_NAME_ correctly depending on the OS. What if other code uses 'OC_BINARY_NAME' in a different context. They will run into the same issue.